PROJECT

```diff
@@ -5,6 +5,9 @@ resources:
 - group: dependency
   kind: KeplerDependency
   version: v1
+- group: dependency
+  kind: KeplerIntegration
```

Comment:
maybe `KeplerGrafanIntegration` (in case there are other types of dashboard to integrate)

Reply:
Should we also create a KeplerPrometheusIntegration for ease of use? Currently KeplerIntegration houses a CR that covers both Grafana and Prometheus.

Reply:
sounds good!